website/docs/foundations/colors/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,106 @@
+
+![Image showing different badges with their associated foreground colors](/assets/foundations/color/colors-status-examples.png)
+
+### Border colors
+
+Use border colors for borders on components, containers or dividers.
+
+Some common examples of border colors include:
+
+- `Border / Strong` for secondary Button border
+- `Border / Primary` for Card border or divider
+- `Border / {Status color}` for Alert borders
+
+![Image showing different borders applied on a button, card and alert](/assets/foundations/color/colors-border-examples.png)
+
+### Surface colors
+
+Use surface colors for the background (or surface) of a component or container. 
+
+Some common examples of surface colors include:
+
+- `Surface / Strong` for the neutral Badge.
+- `Surface / Primary` for component containers.
+- `Surface / Faint` for the secondary Button.
+- `Surface / {Status color}` for Alert background.
+
+![Image showing different surface colors being applied to a badge, card, button and alert](/assets/foundations/color/colors-surface-examples.png)
+
+### Page colors
+
+Page colors are used for page backgrounds. Our components do not use these tokens, however, we recommend `Page / Primary` as the primary background color and `Page / Faint` as a means to create a secondary level on the page for highlighting information, if necessary.
+
+### Disabled colors
+
+[According to WCAG](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html), contrast ratios are only necessary for active elements, therefore, disabled colors do not pass color contrast requirements. Even though we [do not recommend the usage of disabled elements](/patterns/disabled-patterns), there may be instances where they are needed.
+
+In rare instances where token usage is necessary outside of the context of a component, following these standards ensures consistency across our UI:
+- `Foreground / Disabled` for disabled text, icons or other foreground elements
+- `Border / Primary` for disabled borders
+- `Surface / Interactive-Disabled` for disabled backgrounds on interactive elements
+
+![Image showing a disabled input with all the associated semantic tokens](/assets/foundations/color/colors-disabled-examples.png)
+
+## Accessible color combinations
+
+We intend to be conformant with WCAG 2.2 Level AA requirements. In terms of color contrast, this means a luminosity ratio of 4.5:1 for normal sized text, and 3:1 for large text. Further details are outlined on [WCAG’s understanding of Contrast (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html). By default, semantic color tokens provide accessible color combinations out of the box with their associated naming conventions, while using palette colors requires manual validation, especially if you plan to mix and match. As an example, if a color has a semantic status name in it, then other associated status colors will be accessible.
```

Comment:
Nitpicky: should we be more explicit about what "normal" and "large" mean here? Maybe not since we direct users to read more on the w3c site but in my experience some designers assume 16px is "large" and some are unsure where the line is

Reply:
I can add in parenthesis for "normal sized text" to say commonly 16px